### PR TITLE
Fix/i18n errors

### PR DESCRIPTION
**Pull request by nicolaskempf57**
Close https://github.com/etalab/data.gouv.fr/issues/903

We could refactor our locale handling in a new PR to lazy load the messages only for the current lang : https://vue-i18n.intlify.dev/guide/advanced/lazy.html

This also remove `vue3-datepicker`, an unnecessary dependency since https://github.com/etalab/udata-front/pull/95.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -60,8 +60,7 @@
     "process": "^0.11.10",
     "rimraf": "^3.0.2",
     "stylemark": "^3.1.7",
-    "vue-i18n-extract": "^2.0.4",
-    "vue3-datepicker": "^0.3.2"
```
Comment (abulte) [inline package.json:64]: Is this related to this PR?

Comment (nicolaskempf57) [reply package.json:64]: Not really, it was unused since #95

I can rollback and do this in a new PR.

Comment (abulte) [reply package.json:64]: No need I think, a message on the PR should be enough!

**Review by abulte — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (nicolaskempf57) [reply package.json:64]: Done !
